bundle/src/main/java/com/adobe/acs/commons/mcp/impl/DialogResourceProviderFactoryImpl.java

```diff
@@ -21,205 +21,46 @@
 
 import com.adobe.acs.commons.mcp.DialogResourceProviderConfiguration;
 import com.adobe.acs.commons.mcp.DialogResourceProviderFactory;
-import com.adobe.acs.commons.mcp.form.DialogProvider;
```

Comment:
While this is being deprecated - why are you deleting the implementation details now, rather than just waiting for it to be removed in 5.x?

Reply:
yeah, the whole thing is emptied out and the unit tests even confirm that the methods do nothing.

Reply:
Right, but _why_? Sorry i had typo in original question

Reply:
I took out the implementation because I don't want it firing off -- that OSGi service was causing a lot of churn when deploying other model classes (even when disabled!) so the whole point of this exercise was to put that to an end once and for good.  If anyone was calling that service directly (I don't know why they would) then at least it gives them a chance to realize it's going away and they shouldn't try to call it prior to its complete removal.

Reply:
Another possibility (me overlooking the obvious, which happens more than I care to admit) is I could just kill the impl class entirely because it's not part of the public API surface and not a breaking change either.  Would that be more reasonable?

Reply:
Ah ok, no this is good. I was looking for clarity on the reasoning.